test/integration/importA-la-carte.spec.js

```diff
@@ -0,0 +1,130 @@
+import VMenu from '../../src/components/VMenu'
+import VMessages from '../../src/components/VMessages'
+import VNavigationDrawer from '../../src/components/VNavigationDrawer'
+import VOverflowBtn from '../../src/components/VOverflowBtn'
+import VPagination from '../../src/components/VPagination'
+import VParallax from '../../src/components/VParallax'
+import VPicker from '../../src/components/VPicker'
+import VProgressCircular from '../../src/components/VProgressCircular'
+import VProgressLinear from '../../src/components/VProgressLinear'
+import VRadioGroup from '../../src/components/VRadioGroup'
+import VRangeSlider from '../../src/components/VRangeSlider'
+import VSelect from '../../src/components/VSelect'
+import VSlider from '../../src/components/VSlider'
+import VSnackbar from '../../src/components/VSnackbar'
+import VSpeedDial from '../../src/components/VSpeedDial'
+import VStepper from '../../src/components/VStepper'
+import VSubheader from '../../src/components/VSubheader'
+import VSwitch from '../../src/components/VSwitch'
+import VSystemBar from '../../src/components/VSystemBar'
+import VTabs from '../../src/components/VTabs'
+import VTextarea from '../../src/components/VTextarea'
+import VTextField from '../../src/components/VTextField'
+import VTimePicker from '../../src/components/VTimePicker'
+import VToolbar from '../../src/components/VToolbar'
+import VTooltip from '../../src/components/VTooltip'
+import Transitions from '../../src/components/transitions'
+import Vuetify from '../../src/components/Vuetify'
+
+describe('a-la-carte import', () => {
+  Vue.use(Vuetify, {
+    components: {
+      VAlert,
+      VApp,
+      VAutocomplete,
+      VAvatar,
+      VBadge,
+      VBottomNav,
+      VBottomSheet,
+      VBreadcrumbs,
+      VBtn,
+      VBtnToggle,
+      VCard,
+      VCarousel,
+      VCheckbox,
+      VChip,
+      VCombobox,
+      VCounter,
+      VDataIterator,
+      VDataTable,
+      VDatePicker,
+      VDialog,
+      VDivider,
+      VExpansionPanel,
+      VFooter,
+      VForm,
+      VGrid,
+      VIcon,
+      VInput,
+      VJumbotron,
+      VLabel,
+      VList,
+      VMenu,
+      VMessages,
+      VNavigationDrawer,
+      VOverflowBtn,
+      VPagination,
+      VParallax,
+      VPicker,
+      VProgressCircular,
+      VProgressLinear,
+      VRadioGroup,
+      VRangeSlider,
+      VSelect,
+      VSlider,
+      VSnackbar,
+      VSpeedDial,
+      VStepper,
+      VSubheader,
+      VSwitch,
+      VSystemBar,
+      VTabs,
+      VTextarea,
+      VTextField,
+      VTimePicker,
+      VToolbar,
+      VTooltip,
+      Transitions
+    }
+  })
+  const registeredComponents = Object.keys(Vue.options.components).sort()
+  it('should register all subcomponents', () => {
+    expect(registeredComponents).toMatchSnapshot()
+  })
+  it('should register all names in PascalCase', () => {
+    registeredComponents.forEach(name =>
+      expect(name).toMatch(/^(?:[A-Z][a-z]*)+$/)
+    )
+  })
```

Comment:
Separate cases with a blank line, and add one that makes sure everything has been imported here.

```js
// Something like
import * as allComponents from '@/components'
const components = {
  VAlert,
  VApp,
  ...
}
expect(components).toEqual(allComponents)
```

Or maybe forget the separate imports and just do this instead? Everything should be exported from there anyway. 
```js
import * as components from '@/components'`
Vue.use(Vuetify, { components })
```

Reply:
Doubt, then this test would just repeat what the code inside the full import install already does.

Reply:
The point is that otherwise we *will* forget to add new components to the test. I don't care how you do it (well I do but who cares), just make sure something will fail if a new component is made without adding it here.